src/manifestoo/main.py

```diff
@@ -225,10 +226,23 @@ def list(
         None,
         help="Separator character to use (by default, print one item per line).",
     ),
+    sorting: str = typer.Option(
+        "alphabetical",
+        "--sorting",
+        "--sort",
```

Comment:
```suggestion
        "--sort",
```

`--sorting` sounds a bit strange to me. Not sure.